src/main/java/dk/cngroup/lentils/controller/StatisticsController.java

```diff
@@ -0,0 +1,75 @@
+package dk.cngroup.lentils.controller;
+
+import dk.cngroup.lentils.entity.CypherStatus;
+import dk.cngroup.lentils.service.StatisticsService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/game/statistics")
+public class StatisticsController {
+
+    private static final String STATISTICS_VIEW = "statistics/statistics";
+
+    private static final String HINT_TAKEN_COUNT_HEADER = "Seřazeno podle celkového počtu použitých nápověd";
+    private static final String HINT_TAKEN_POINTS_HEADER = "Seřazeno podle celkového počtu bodů za nápovědy";
+    private static final String SKIPPED_TEAMS_COUNT_HEADER = "Seřazeno podle počtu týmů, kteří přeskočili";
+    private static final String SOLVED_TEAMS_COUNT_HEADER = "Seřazeno podle počtu týmů, kteří vyřešili";
+
+    private static final String HINT_TAKEN_COUNT_COLUMN = "Počet nápověd";
+    private static final String HINT_TAKEN_POINTS_COLUMN = "Body za nápovědy";
+    private static final String SKIPPED_TEAMS_COLUMN = "Týmů přeskočilo";
+    private static final String SOLVED_TEAMS_COLUMN = "Týmů vyřešilo";
+
+    @Autowired
```

Comment:
Přepsal bych na injekci do konstruktoru.

Reply:
je to tam ;)